builtin/game/register.lua

```diff
@@ -368,6 +368,11 @@ core.register_node(":ignore", {
 	air_equivalent = true,
 	drop = "",
 	groups = {not_in_creative_inventory=1},
+	-- Prevent placement of ignore (to prevent engine error message spam in logs)
+	on_place = function(itemstack, placer, pointed_thing)
+		minetest.chat_send_player(placer:get_player_name(), minetest.colorize("#FF0000", "One does not simply place ignore nodes in the world!"))
```

Comment:
Whereas I like the meme potential the line is too long and this message is superfluous. You could return an empty stack instead.
Also: `on_place = function() end` would do the same, but without emptying the stack.

Reply:
I agree, the message just replaces spam with spam.